powerdns/models/powerdns.py

```diff
@@ -723,3 +727,39 @@ class Meta:
 
     def __str__(self):
         return self.domain
+
+
+
+try:
+    import pyhermes
+except ImportError:
+    pass
+else:
+    if settings.TXT_AUTOUPDATE_TOPIC_NAME:
```

Comment:
imo no need for this if either; if you don't want to use this endpoint (subscriber), you'll just remove it/suspend it in hermes.